lib/ex_integrate/zipper.ex

```diff
@@ -0,0 +1,82 @@
+defmodule ExIntegrate.Core.Zipper do
```

Comment:
interesting solution RE: zipper. I guess this lets you get more arbitrary than the tools libgraph offers to move around the graph eh?

this makes sense as I see in this context that Libgraph doesn't really have a way to specifically query the graph. It does have algorithms for searching, or algorithmic processing, but nothing that would let you pinpoint positions on it